examples/video_dataset_loading/main.py

```diff
@@ -0,0 +1,109 @@
+from kale.loaddata.video_dataset import  VideoFrameDataset, imglist_totensor
+from torchvision import transforms
+import torch
+import matplotlib.pyplot as plt
+import os
+
+
+if __name__ == '__main__':
+    """
+    This demo uses the dummy dataset inside of the folder "demo_dataset".
+    It is structured just like a real dataset would need to be structured.
+    """
+    videos_root = os.path.join(os.getcwd(), 'demo_dataset')
+    annotation_file = os.path.join(videos_root, 'annotations.txt')
+
+    """ DEMO 1 WITHOUT IMAGE TRANSFORMS """
+    dataset = VideoFrameDataset(
+        root_path=videos_root,
+        annotationfile_path=annotation_file,
+        num_segments=5,
+        frames_per_segment=1,
+        imagefile_template='img_{:05d}.jpg',
+        transform=None,
+        random_shift=True,
+        test_mode=False
+    )
+
+    sample = dataset[0]
+    frames = sample[0]  # list of PIL images
+    label = sample[1]   # integer label
+
+    for image in frames:
+        plt.imshow(image)
+        plt.title(label)
+        plt.show()
+        plt.pause(1)
+
+
+    """ DEMO 2 WITH TRANSFORMS """
+    # As of torchvision 0.8.0, torchvision transforms support batches of images
+    # of size (BATCH x CHANNELS x HEIGHT x WIDTH) and apply deterministic or random
+    # transformations on the batch identically on all images of the batch. Any torchvision
+    # transform for image augmentation can thus also be used  for video augmentation.
+    preprocess = transforms.Compose([
+        transforms.Lambda(imglist_totensor),  # list of PIL images to (FRAMES x CHANNELS x HEIGHT x WIDTH) tensor
+        transforms.Resize(299),  # image batch, resize smaller edge to 299
+        transforms.CenterCrop(299),  # image batch, center crop to square 299x299
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+
+    dataset = VideoFrameDataset(
+        root_path=videos_root,
+        annotationfile_path=annotation_file,
+        num_segments=5,
+        frames_per_segment=1,
+        imagefile_template='img_{:05d}.jpg',
+        transform=preprocess,
+        random_shift=True,
+        test_mode=False
+    )
+
+    sample = dataset[1]
+    # tensor of shape (NUM_SEGMENTS*FRAMES_PER_SEGMENT) x CHANNELS x HEIGHT x WIDTH
+    frame_tensor = sample[0]
+    print('Video Tensor Size:', frame_tensor.size())
+    # integer label
+    label = sample[1]
+
+
+    def denormalize(video_tensor):
+        """
+        Undoes mean/standard deviation normalization, zero to one scaling,
```

Comment:
For what?

Reply:
When you use a dataset class with transforms and then try to access a sample, you receive the sample after the transforms are applied. So, in the beginning each frame of a sample consists of an integer tensor with values betwen 0 and 255, but when you access that sample, each frame is scaled to range [0,1] and then  normalized to mean zero and standard deviation one before you receive it. After that you cannot plot the frames anymore for visualization, because plotters only know how to show images either in the range [0,1] or [0,255].  
  
So, typically you need to denormalize correctly back to [0, 255] or [0,1] before being able to do any visualization like I do below those lines.

Reply:
Thanks! Clarifying this in the docstrings may be better.